contrib/endpoints/src/api_manager/config_manager.h

```diff
@@ -71,13 +73,15 @@ class ConfigManager {
   void Init(RolloutApplyFunction config_rollout_callback);
```

Comment:
This callback is only used by the timer,   it should be set in the constructor.   Please add comment too
Otherwise, it will be very confusing.